### PR TITLE
Add benchmark suite output parser [ci skip]

### PR DESCRIPTION
**Pull request by kripken**
Just a helpful little tool to get something that can be imported
into a spreadsheet.

### CODE REVIEW

File: tools/parse_benchmark_output.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Copyright 2020 The Emscripten Authors.  All rights reserved.
```
Comment (sbc100) [inline tools/parse_benchmark_output.py:1]: #!/usr/bin/env python3

Comment (kripken) [reply tools/parse_benchmark_output.py:1]: Thanks, done.

**Comment by kripken** [comment]
I agree emitting JSON directly from the test runner is a good option too. That would require it to be optional, though, as we do want the current human-readable format by default I think - it's what I use 99% of the time. Not sure we have a good way to pass options through test runner atm for something like that.

For now I think a parser script on the side is the simplest thing, and given the benchmark suite is just an optional/untested thing for our own convenience, not sure it's worth doing more here.

**Comment by sbc100** [comment]
Ok, please put the script in the `tests/` directory though since its not part of the actual toolchain.

**Pull request merged**